### DISCUSSION TITLE
Convert models to timm models

### DISCUSSION THREAD

**Discussion by usuyama** [open-mmlab/mmselfsup#595] [How-to]
Thinking how we can convert the trained models to timm format, so we can evaluate them in other benchmark / pipelines.

**Comment by fangyixiao18** [comment]
Currently in MMSelfSup, we don't have such scripts, you could print our model and their model keys, and write a script to modify them. Thanks~

**Reply by usuyama** [reply]
Thanks for your reply, @fangyixiao18 ! I'm new to MMSelfSup, so good to know the current situation.

Wonder if it's a frequent request/scenario, or it's just me.
